### PR TITLE
feat: add intro text

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Review by ojeytonwilliams — DISMISSED** [review]
LGTM 👍

File: client/i18n/locales/english/intro.json [hunk 1]
```diff
@@ -77,28 +77,34 @@
     "blocks": {
       "build-a-tribute-page-project": {
         "title": "Build a Tribute Page Project",
-        "intro": ["placeholder", "holder of place"]
-      },
+        "intro": [
+          "This is one of required projects to earn your certification.",
```
Comment (moT01) [inline client/i18n/locales/english/intro.json:81]:
```suggestion
          "This is one of the required projects to earn your certification.",
```

File: client/i18n/locales/english/intro.json [hunk 2]
```diff
@@ -77,28 +77,34 @@
...
+          "For this project, you will build a tribute page for a subject of your choosing, fictional or real."
+        ] },
       "build-a-personal-portfolio-webpage-project": {
         "title": "Build a Personal Portfolio Webpage Project",
-        "intro": ["placeholder", "holder of place"]
-      },
+        "intro": [
+          "This is one of required projects to earn your certification.",
```
Comment (moT01) [inline client/i18n/locales/english/intro.json:87]:
```suggestion
          "This is one of the required projects to earn your certification.",
```

File: client/i18n/locales/english/intro.json [hunk 3]
```diff
@@ -77,28 +77,34 @@
...
+          "For this project, you will build your own personal portfolio page."
+        ] },
       "build-a-product-landing-page-project": {
         "title": "Build a Product Landing Page Project",
-        "intro": ["placeholder", "holder of place"]
-      },
+        "intro": [
+          "This is one of required projects to earn your certification.",
```
Comment (moT01) [inline client/i18n/locales/english/intro.json:93]:
```suggestion
          "This is one of the required projects to earn your certification.",
```

File: client/i18n/locales/english/intro.json [hunk 4]
```diff
@@ -77,28 +77,34 @@
...
+          "For this project, you will build a product landing page to market a product of your choice."
+        ] },
       "build-a-survey-form-project": {
         "title": "Build a Survey Form Project",
-        "intro": ["placeholder", "holder of place"]
-      },
+        "intro": [
+          "This is one of required projects to earn your certification.",
```
Comment (moT01) [inline client/i18n/locales/english/intro.json:99]:
```suggestion
          "This is one of the required projects to earn your certification.",
```

File: client/i18n/locales/english/intro.json [hunk 5]
```diff
@@ -77,28 +77,34 @@
...
       "build-a-technical-documentation-page-project": {
         "title": "Build a Technical Documentation Page Project",
-        "intro": ["placeholder", "holder of place"]
-      },
-      "learn-css-colors-by-building-a-set-of-colored-markers": {
-        "title": "Learn CSS Colors by Building a Set of Colored Markers",
-        "intro": ["placeholder", "holder of place"]
-      },
+        "intro": [
+          "This is one of required projects to earn your certification.",
```
Comment (moT01) [inline client/i18n/locales/english/intro.json:105]:
```suggestion
          "This is one of the required projects to earn your certification.",
```

**Review by moT01 — COMMENTED** [review]
minor grammar issue

**Review by raisedadead — APPROVED** [review]

**Review by moT01 — APPROVED** [review]

**Pull request merged**